README.md

```diff
@@ -468,6 +468,18 @@ We appreciate your contribution to the development of this project!
 
 [![GitHub contributors (via allcontributors.org)](https://contrib.rocks/image?repo=cunarist/rust-in-flutter)](https://github.com/cunarist/rust-in-flutter/graphs/contributors)
 
+# 👏 Thanks
+
+These are tools for connecting Flutter and native compilers.
+
+Rust-In-Flutter was not made alone. There were pioneers that gave inspiration to the structure of this package. Credits to these wonderful efforts!
+
+- https://github.com/fzyzcjy/flutter_rust_bridge
+- https://github.com/superlistapp/super_native_extensions
+- https://github.com/brickpop/flutter-rust-ffi
+- https://github.com/corrosion-rs/corrosion
+- https://github.com/irondash/cargokit
+
```

Comment:
@temeddix Because the folder `connectors` has been deleted in this branch, the content of `connectors/README.md` has been move to here, how do you think about it? Do you have any better thoughts?

Reply:
I personally think it would be better to move this to 'discussions' in Github in 'announcements' category. I will also handle this :)

Reply:
https://github.com/cunarist/rust-in-flutter/discussions/104

It's now written here